File: CodeComp.2015.Nov.V1/SourceCode/SampleC#Client/Strategy/ICardStrategy.cs
using System.Collections.Generic;
using RBS.CodeComp.Client.Entity;

namespace RBS.CodeComp.Client.Strategy
{
    internal interface ICardStrategy
    {
        List<Card> PassCards(GameStatus gameStatus);

        Card PlayCard(GameStatus gameStatus, string myTeamName);
    }
}

File: src/main/java/com/mgs/rbsnov/client/interfaces/ICardStrategy.java
package com.mgs.rbsnov.client.interfaces;

import java.util.List;

import com.mgs.rbsnov.client.entities.Card;
import com.mgs.rbsnov.client.entities.GameStatus;

public interface ICardStrategy {
	List<Card> PassCards(GameStatus gameStatus);

    Card PlayCard(GameStatus gameStatus, String myTeamName);
}
